### ISSUE TITLE
Loinc 48416-2:  "Alpha-1-acid glycoprotein [Mass/volume] in Body fluid"

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4632]
Suggest creating a new term for Loinc: 48416-2 [Alpha-1-acid glycoprotein [Mass/volume] in Body fluid] 
New term label:Abnormal circulating Alpha-1-acid glycoprotein level
New term comment (if any):Child term Increase/Decrease circulating Alpha-1-acid glycoprotein level ;  Parent term Abnormal circulating protein level HP:0010876
Your biocurator ID for loinc2hpo (if desired): OHSU:jpgourdine

**Comment by pnrobinson** [comment]
see this https://www.sciencedirect.com/science/article/pii/S0167483800001539

**Comment by pnrobinson** [comment]
In think this metabolite is measured in urine, and there is a term for this
https://hpo.jax.org/app/browse/term/HP:0032377
We should not create HPO terms for LOINCs that refer to bodily fluid (too general!).
